### PR TITLE
PXF CI: run pxf_automation suite as part of pxf regression test job

### PR DESCRIPTION
**Pull request by denalex**

### CODE REVIEW

**Comment by denalex** [comment]
To be ported to 5X_STABLE as well.

**Comment by sansanichfb** [comment]
Should we keep both pxf_automation and pxf regression tests or we can just use pxf_automation?

**Comment by denalex** [comment]
we will remove regression tests later once we port more HDFS tests (now only smoke is running)

File: concourse/scripts/regression_tests_pxf.bash [hunk 1]
```diff
@@ -36,15 +37,44 @@ function gen_env(){
...
+	cat > /home/gpadmin/run_pxf_automation_test.sh <<-EOF
+	set -exo pipefail
+
+	source /usr/local/greenplum-db-devel/greenplum_path.sh
+	source \${1}/gpdb_src/gpAux/gpdemo/gpdemo-env.sh
+
+	export PG_MODE=GPDB
+	export GPHD_ROOT=\${1}/singlecluster
+
+    cp -r /usr/lib64/python2.6/site-packages/psi $GPHOME/lib/python
```
Comment (shivzone) [inline concourse/scripts/regression_tests_pxf.bash]: Can we always assume python 2.6 will be the version in CI ?

Comment (Eulerizeit) [reply concourse/scripts/regression_tests_pxf.bash]: No. Ideally we will be moving to whatever Python is on the system. Currently only RHEL 6 has 2.6 and I think we are actually vendoring 2.7 for the enterprise build.

Comment (shivzone) [reply concourse/scripts/regression_tests_pxf.bash]: This python module is needed only to support the gpdb-pxf automation tests that uses tinc. These automation tests are yet to be shipped with the gpdb source (they will be done eventually, but by then tinc wouldn't be used anymore).
Alternate option is to use the system pip to install this module into a custom location. eg: pip install --ignore-installed --install-option="--prefix=/tmp" psi

File: concourse/scripts/regression_tests_pxf.bash [hunk 2]
```diff
@@ -36,15 +31,46 @@ function gen_env(){
...
+		ln -s ${PXF_HOME}/conf conf
+		for X in ${PXF_HOME}/lib/pxf-*-[0-9]*.jar; do \
+			ln -s ${X} $(echo $(basename ${X}) | sed -e 's/-[a-zA-Z0-9.]*.jar/.jar/'); \
+		done
+	popd > /dev/null
+
+	cat > /home/gpadmin/run_pxf_automation_test.sh <<-EOF
+	set -exo pipefail
+
+	source /usr/local/greenplum-db-devel/greenplum_path.sh
```
Comment (sansanichfb) [inline concourse/scripts/regression_tests_pxf.bash]: Maybe use GPHOME instead.

Comment (lavjain) [reply concourse/scripts/regression_tests_pxf.bash]: GPHOME is set by running that script

**Pull request merged**